libnavigation-ui/src/main/res/values/styles.xml

```diff
@@ -93,18 +93,18 @@
             @color/mapbox_navigation_view_color_progress_background
         </item>
 
-        <item name="navigationViewRouteStyle">@style/NavigationMapRoute</item>
+        <item name="navigationViewRouteStyle">@style/MapboxStyleNavigationMapRoute</item>
 
-        <item name="navigationViewLocationLayerStyle">@style/NavigationLocationLayerStyle</item>
+        <item name="navigationViewLocationLayerStyle">@style/MapboxStyleNavigationLocationLayerStyle</item>
 
-        <item name="navigationViewDestinationMarker">@drawable/map_marker_light</item>
+        <item name="navigationViewDestinationMarker">@drawable/mapbox_ic_map_marker_light</item>
```

Comment:
Could we convert this into a Vector Drawable so we can remove the PNGs? cc @map-angela

Reply:
same as above.